tests/integration/targets/vmware_ops_esxi_maintenance_mode_test/vars/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+# General vars
+resource_prefix: host-maintenance 
+resource_pool_name: "{{ resource_prefix }}-pool"
+cluster_name: "Eco-Cluster"
+datacenter_name: "Eco-Datacenter"
+
+# Vars for cloud.vmware_ops.provision_virtual_esxi role
+provision_virtual_esxi_hostname: "{{ vcenter_hostname }}"
+provision_virtual_esxi_username: "{{ vcenter_username }}"
+provision_virtual_esxi_password: "{{ vcenter_password }}"
+provision_virtual_esxi_validate_certs: false
+provision_virtual_esxi_port: 443
+provision_virtual_esxi_cluster: "{{ cluster_name }}"
+provision_virtual_esxi_datacenter: "{{ datacenter_name }}"
+provision_virtual_esxi_folder: ""
+provision_virtual_esxi_resource_pool: "{{ resource_pool_name }}"
+provision_virtual_esxi_datastore_iso_path: "[eco-nfs-datastore-iso] custom_esxi_8.iso"
+provision_virtual_esxi_vms:
+  - name: ci-vcenter-maintenance-test
+network_name: "TestNetwork"
```

Comment:
It is not a role variable. Maybe move it to the General vars.

Reply:
ok